core-initrd/latest/factory/usr/lib/udev/rules.d/90-ubuntu-core-partitions.rules

```diff
@@ -0,0 +1,15 @@
+SUBSYSTEM!="block", GOTO="ubuntu_core_partitions_end"
+
+ENV{DEVTYPE}=="partition", IMPORT{parent}="UBUNTU_*_UUID"
+IMPORT{program}="/usr/lib/snapd/snap-bootstrap scan-disk"
+ENV{DEVTYPE}=="disk", ENV{UBUNTU_DISK}=="1", SYMLINK+="disk/ubuntu/disk"
```

Comment:
I wonder if it's a good idea to use just ubuntu there or if we risk to clash with something done by another team in this space

Reply:
"snapd" then?

Reply:
I suppose so